tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -71,12 +71,23 @@ def parameters(cls):
 
         return parameters
 
+    def check_params(self):
```

Comment:
not clear name for me... Why `check_params` check only part of params and we have additional checks in `configure`?

Besides that, I think it is more logical to separate the realization selection logic than just check of this parameter

Reply:
Added new functions for checking parameters